app/assets/stylesheets/components/tooltip.scss

```diff
@@ -0,0 +1,84 @@
+.tooltipped {
+  position: relative;
+}
+.tooltip {
+  font-size: 0.8rem;
+  text-align: center;
+  font-style: normal;
+  position: absolute;
+  left: 50%;
+  max-width: 150px;
+  background: $bright-gray;
+  color: $white;
+  border-radius: 2px;
+  -webkit-border-radius: 2px;
```

Comment:
is it a good idea to import the tachyons classes for border-radius and other properties, or is it better to just create custom ones? 

how do you determine whether to use the table of styles to look up what you need, or to just write it in?

Reply:
Yeah there's no set rule here. For the border radius we should indeed have a single value that we use everywhere. Tachyons itself doesn't provide though, it's totally agnostic